README.md

```diff
@@ -291,7 +291,7 @@ As a starting point, we recommend visualising sociotechnical architecture with t
 
 #### Tools
 
-- [Dynamic Reteaming](https://leanpub.com/dynamicreteaming) 
+- [Dynamic Re-teaming](https://leanpub.com/dynamicreteaming) 
```

Comment:
@Andrea, the book is called "Dynamic Reteaming", so I would suggest we keep that name here.